pkg/api/v1/control-plane/resource/envoy_resource.go

```diff
@@ -1,11 +1,9 @@
 package resource
 
 import (
-	envoy_api_v2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
```

Comment:
Deleted v2 API resources from here. Confirmed that Gloo is no longer passing in v2 resources